source/docs/07-tablature.xml

```diff
@@ -96,50 +103,79 @@
       <body>
          <div xml:id="tablature" type="div1">
             <head>Repertoire: String Tablature</head>
-            <p>This chapter describes the attribute classes that are part of the MEI.tablature module.</p>
-            <div xml:id="tablatureOverview" type="div2">
-               <head>Overview of the Tablature Module</head>
-               <p>The tablature module is used to record basic tablature notation. It is designed primarily for guitar and similar plucked-string instruments.</p>
-               <p>The <att>lines</att> attribute on the <gi scheme="MEI">staffDef</gi> element is used to define the number of lines, courses, or strings, present in the tablature. The <att>tab.strings</att> attribute is then used to enumerate the pitches of the open strings. It is important to note that this is given using the written pitch, not the sounding pitch. For example, the Western 6-string guitar, in standard tuning, sounds an octave below written pitch.</p>
-               <p>The <att>tab.strings</att> attribute gives the string tuning, ordered from highest to lowest pitch.</p>
-               <p>For standard guitar tuning, the <gi scheme="MEI">staffDef</gi> element might look like this:</p>
+            <p>This chapter describes the attribute classes that are part of the MEI.stringtab
+               module (which will be renamed to MEI.tablature in a future version).</p>
+            <p>The tablature module is used to record basic tablature notation. It is designed
+               primarily for guitar and similar plucked-string instruments.</p>
+
+            <div xml:id="tablature-def" type="div2">
+               <head>Defining Tablature Parameters</head>
+               <p>The <att>lines</att> attribute on the <gi scheme="MEI">staffDef</gi> element is
+                  used to define the number of lines, courses, or strings, present in the tablature.
+                  The <att>tab.courses</att> attribute is then used to enumerate the pitches of the
```

Comment:
I'm not sure what to believe here. It says to use `@tab.courses`, but when I look at [the definition](https://github.com/music-encoding/music-encoding/pull/1468/files#diff-5c074868b25be0067795e47d9c42c1f58959c7e986419fec3f774ccd677196e9R128) of `@tab.courses` it says it's deprecated in favour of the `<tuning>` element?

Reply:
Thanks. Guidelines docs have now been rewritten, and `@tab.courses` removed